static/test/javascripts/spec/common/adblock-banner.spec.js

```diff
@@ -0,0 +1,22 @@
+define([
+    'common/modules/commercial/adblock-banner-config',
+    'common/modules/adblock-banner'
+], function (adblockBannerConfig, AdblockBanner) {
+
+    describe('Adblock banner', function () {
+
+        var ukBanners;
+        beforeEach(function () {
+            document.innerHtml = '<div class="top-banner-ad-container--desktop"></div>';
+            ukBanners = adblockBannerConfig.getBanners('UK');
+        });
+
+        it('should be able to render every type of banner', function () {
```

Comment:
This test is probably a bit misleading in that what I'm actually testing is that the configs for the templates provide all the variables etc that the templates need. 

I could probably include it alongside all the other config tests and name it slightly better.